src/lib.rs

```diff
@@ -118,7 +113,7 @@ impl GeoHash {
         }
 
         let two = I64F64::from_num(2);
-        while out.len() < len {
+        for byte in &mut out {
             for _ in 0..5 {
```

Comment:
@pifragile I have problems understand the `0..5` bound. Can you explain that to me?

Reply:
looks like hard-coding as 5 is the number of chars we use for cid, right?

Reply:
I don't think so (and hope not) because it has nothing to do with the length of the returned array.

If it is indeed related, I would probably screw up all calculations for lengths != 5. But as there are tests for other lengths, I think it is unrelated.